<?xml version="1.0"?>
<launch>
    <!--Launch Realsense-->
    <include file="$(find realsense2_camera)/launch/realsense0.launch"/>

    <!--object detection-->
    <node pkg="tensorflow_object_detection_tx2" type="tensorflow_object_detection_ros.py" name="tensorflow_object_detection_realsense0" output="screen" >
        <remap from="/image" to="/camera0/color/image_raw" />
        <remap from="/object_image" to="/object_image/0" />
        <remap from="/object_info"  to="/object_info/0" />
    </node>
</launch>
